File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.cpp
class Solution {
public:
    vector<int> targetIndices(vector<int>& nums, int target) {
        vector<int> find;
        int l=nums.size();
        int temp;
        /*for(int i=0; i<l - 1; i++){
            for(int j=0; j<l-i-1; j++){
          if(nums[j] > nums[j+1]){
             temp =nums[j];
             nums[j] =nums[j+1];
             nums[j+1] = temp;
          }
        }
        }*/
        sort(nums.begin(),nums.end());
        for(int i=0; i<nums.size(); i++){
            if(target !=0 && nums[i]==target){
                find.push_back(i);
            }
        }
        return find;
    }
};

File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        nums.sort()
        # bottom = 0
        # top = length - 1
        answer = []
        for index in range(length):
            if nums[index] == target:
                answer.append(index)
                
        return answer